src/Pagination/Pagination.stories.svelte

```diff
@@ -0,0 +1,97 @@
+<script context="module">
+  import Pagination from './Pagination.svelte';
+
+  export const meta = {
+    title: 'Stories/Pagination',
+    component: Pagination,
+    parameters: {},
+    argTypes: {
+      class: {
+        className: 'string',
+        table: { disable: true }
+      },
+      listClassName: {
+        control: 'text'
+      },
+      size: {
+        control: {
+          type: 'select'
+        },
+        options: ['', 'sm', 'lg']
```

Comment:
```suggestion
        options: ['default', 'sm', 'lg']
```
could call this default for the examples